scripts/repl.js

```diff
@@ -1,5 +1,15 @@
 (function(babel, $, _, ace, window) {
 
+  /* Throw meaningful errors for getters of commonjs. */
+  ["module", "exports", "require"].forEach(function(commonVar){
+    Object.defineProperty(window, commonVar, { 
+      get: function () { 
+      throw new Error(commonVar + " is not supported in the browser,\
+        you need a commonjs environment such as node.js/io.js, browserify/webpack etc");
```

Comment:
@thejameskyle Oh, it looked fine in the console, do you want the entire message in a single line?  As in with out the `\` 

Oh man it would awesome if we rewrite this in ES6, we could have used template string ;)